docs/plugins/verifier/notation.md

```diff
@@ -260,3 +260,19 @@ Sample Notation CLI config with timestamping configuration:
     }
 }
 ```
+
+## Certificate Revocation Check (CRL)
+
+Ratify supports CRL based on notation-core-go v1.2.0-rc1 and notation-go v1.3.0-rc1
+
+## Intro
```

Comment:
Does Ratify supports both Base CRL and Delta CRL?

Reply:
Delta CRL support seems to be under developing: https://github.com/notaryproject/notation-core-go/pull/247/files#diff-3d2a68889bf60d46652f995e5c68976f12d7e777865a6723f3209c5add8dc118

Need to check with notation team for further details.

Reply:
For more details, please refer to this issue: https://github.com/notaryproject/notation-core-go/issues/228